core/src/io/github/eng12020team24/project1/Button.java

```diff
@@ -0,0 +1,74 @@
+package io.github.eng12020team24.project1;
+
+import com.badlogic.gdx.Gdx;
+//import com.badlogic.gdx.Game;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+
+
+public class Button{
+    private Integer x;
+    private Integer y;
+    Integer width;
+    Integer height;
+    private TextureRegion image;
+    //Game game;
+    public Button(Integer x, Integer y, TextureRegion image){
+        this.x=x;
+        this.y=y;
+        this.width=image.getRegionWidth();
+        this.height=image.getRegionHeight();
+        this.image=image;
+        //this.game=game;
+    }
+    /**
+     * Returns if this button has been clicked on by finding if a click has occured whilst the mouse is over this button
+     * @return if this button has been clicked
+     */
+    public boolean isClicked(){
+        if (Gdx.input.isTouched()){
+            if(isMouseOver()){
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Returns if the mouse is currently positioned over this button
+     * @return if mouse currently positioned over this button
+     */
+    public boolean isMouseOver(){
+        if (Gdx.input.getY()>Gdx.graphics.getHeight()-y-height 
+        && Gdx.input.getY()<Gdx.graphics.getHeight()-y 
```

Comment:
This isn't exactly clear to me?  Why not do something like `Gdx.graphics.getHeight() - Gdx.input.getY() - 1` which is equivalent but IMO a bit easier to read.